recipes/rsatoolbox/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "rsatoolbox" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rsatoolbox-{{ version }}.tar.gz
+  sha256: 2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213
+
+build:
+  skip: true  # [py<37 or py>310 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython # >=3.0.0a11
+    - numpy >=1.21.2
```

If this package doesn't link to numpy by compiling code, then you don't need numpy in host. The same is true for scipy.

---

In other words, assuming you're compiling against numpy's C-API
```suggestion
    - numpy
```

If not, remove

---

Thank you @beckermr and @h-vetinari !! I don't have much experience with conda but these all seem like good suggestions and I've implemented them.

Having a closer look at our Cython code it turns out we don't currently have a build-time dependency on `numpy` (it returns regular memory views which are then converted to numpy arrays in the calling code). We do depend on scipy (`blas.dgemv`):

https://github.com/rsagroup/rsatoolbox/blob/bef1775960773e39ed6e9487047e298ad0b25542/src/rsatoolbox/cengine/similarity.pyx#L8

So I've switched out the host requirements but got rid of the `pin_compatible()` restraint. 

Thanks again and let me know if there's anything else to fix